src/__tests__/components/common/Label/index.jsx

```diff
@@ -1,9 +1,9 @@
 import React from 'react';
 import { shallow } from 'enzyme';
 import context from 'jest-context';
-import '../../../../setupTests';
+import 'SetupTests';
 
-import Label from '../../../../components/common/Label';
+import Label from 'Common/Label';
```

Comment:
You can create an index in common root to export label, then use it like:

import { Label } from 'Common';